Makefile

```diff
@@ -39,7 +39,7 @@ JSSTYLE_FLAGS    = -o "indent=2,doxygen,unparenthesized-return=0,line-length=120
 REPO_MODULES	 =
 SMF_MANIFESTS_IN = smf/manifests/adminui.xml.in
 
-NODE_PREBUILT_VERSION=v0.10.45
+NODE_PREBUILT_VERSION=v4.4.7
```

Comment:
We should probably be moving to `v0.12.15` for now, as that's the latest build of `sdcnode` that [appears to exist](https://download.joyent.com/pub/build/sdcnode/fd2cc906-8938-11e3-beab-4359c665ac99/master-latest/sdcnode/)